### DISCUSSION TITLE
part-ii/2021-132/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#380] [Question comments]
Part II 2021 Representation Theory: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2021-132/

**Comment by Kiolt** [comment]
<details>
<summary>(a)</summary>

Show that if the statement is false the kernel of the Vandermode matrix is non-trivial.

</details>

<details>
<summary>(b)</summary>

Characters span, but indicators on class functions distinguish between $g$ and $h$.

</details>

<details>
<summary>(c)</summary>

Let $\chi(g)=\chi(e)$. Note that for any irred char $|\psi(g)|\le|\psi(e)|$, so any $\psi$ which distinguishes between $g$ and $e$ works by the triangle inequality.

</details>
